src/server/tests/game.test.ts

```diff
@@ -99,7 +99,7 @@ describe("Turn tests", () => {
 
     // You should be able to select all of a team's words without having to end the turn
     for (let i = 0; i < wordsPerTeam; i += 1) {
-      const val = `${i}-team1`;
+      const val = `${strFromNum(i)}-team-one`;
```

Comment:
Modifying this test because digits are no longer allowed in words with the validation